example-pipelines/environments/Jenkinsfile

```diff
@@ -50,7 +60,12 @@ pipeline {
       when { anyOf {branch "prod";branch "dev" } }
       steps {
         container('terraform') {
-          sh '''cd example-pipelines/example-create
+          sh '''echo ""
+          echo "*************** TERRAFOM APPLY ******************"
+          echo "******* At environment: ${CHANGE_TARGET} ********"
```

Comment:
This will only run on the origin/dev or orgin/prod branch and not a PR. In that case will $CHANGE_TARGET be present?

Reply:
Ah I missed this. Need to test the merge scenario for this.

Reply:
Done. So $CHANGE_TARGET is only present and useful when creating a PR. $BRANCH_NAME only works when merging a PR. Updated Jenkinsfile accordingly.